[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@cc09e11 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@636f2863 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@6b571209 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@1d6cd023 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@466e83a6 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@5cd89738 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@f4c11f0 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@745a25f2 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@461beb6c (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:02:56] [User Authenticator #3/WARN]: [User Authenticator #3] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@5e62d5c0 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[01:03:00] [User Authenticator #3/INFO]: UUID of player Indominux is 9fe6f167-3d29-33e1-9d7d-983eb4300bbc
[01:03:00] [Server thread/INFO]: Indominux joined the game
[01:03:00] [Server thread/INFO]: Indominux[/179.24.142.187:41009] logged in with entity id 4 at ([world]1.3973069313844464, 143.0, 9.566801921811514)
[01:03:10] [Craft Scheduler Thread - 21/INFO]: [AuthMe] Indominux logged in 179.24.142.187
[01:03:16] [Server thread/INFO]: Indominux lost connection: Disconnected
[01:03:16] [Server thread/INFO]: Indominux left the game
[05:34:55] [Craft Scheduler Thread - 76/INFO]: [Vault] Checking for Updates ... 
[05:34:59] [Craft Scheduler Thread - 76/INFO]: [Vault] No new version available
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@431ef2c0 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@49e699b0 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@ce41ee8 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@38586546 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@3da29112 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@40f4b40f (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@6ccbe1a7 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@1742db2d (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@3ec1f2d6 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:22] [User Authenticator #4/WARN]: [User Authenticator #4] WARN fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase - AuthMeMYSQLPool - Failed to validate connection fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl@5bf07dcd (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
[09:16:26] [User Authenticator #4/INFO]: UUID of player alemarsiano is 0337aff9-4c8d-3444-a5d9-11317ddae49c
[09:16:26] [Server thread/INFO]: alemarsiano joined the game
[09:16:26] [Server thread/INFO]: alemarsiano[/167.61.150.60:40235] logged in with entity id 5 at ([world]1.3973069313844464, 143.0, 9.566801921811514)
[09:16:48] [Server thread/INFO]: [AuthMe] alemarsiano registered 167.61.150.60
[09:16:48] [Craft Scheduler Thread - 123/INFO]: [AuthMe] alemarsiano logged in 167.61.150.60
[09:16:53] [Server thread/INFO]: alemarsiano lost connection: Disconnected
[09:16:53] [Server thread/INFO]: alemarsiano left the game
[11:34:55] [Craft Scheduler Thread - 150/INFO]: [Vault] Checking for Updates ... 
[11:34:59] [Craft Scheduler Thread - 150/INFO]: [Vault] No new version available
[17:34:55] [Craft Scheduler Thread - 222/INFO]: [Vault] Checking for Updates ... 
[17:34:59] [Craft Scheduler Thread - 222/INFO]: [Vault] No new version available
[23:34:55] [Craft Scheduler Thread - 294/INFO]: [Vault] Checking for Updates ... 
[23:34:59] [Craft Scheduler Thread - 294/INFO]: [Vault] No new version available
